src/content/docs/get-started/index.mdx

```diff
@@ -178,10 +178,10 @@ Make sure you are signed into your GitHub account before you start. Otherwise, y
 
 <Callouts columnCount="1">
 
-1. Navigate to <Link href="https://github.com/hlxsites/boilerplate-commerce-dropins" text="boilerplate-commerce-dropins" />, select the **Use this template** button, then select the **Create a new repository** option to open the form.
+1. Navigate to <Link href="https://github.com/hlxsites/aem-boilerplate-commerce" text="aem-boilerplate-commerce" />, select the **Use this template** button, then select the **Create a new repository** option to open the form.
 1. Complete the form with the following details:
 
-   - **Repository template**: `hlxsites/boilerplate-commerce-dropins` (default).
+   - **Repository template**: `hlxsites/aem-boilerplate-commerce` (default).
    - **Include all branches**: Do not include all branches (default).
```

Comment:
This is fine, and I think we can potentially include some manual instructions for checking the specific `develop` branch from the boilerplate, if desired.

Reply:
Yeah, now that I think of it, we could probably add steps to the [local development](https://experienceleague.adobe.com/developer/commerce/storefront/get-started/#set-up-local-dev-environment) section at the end of the tutorial.